scss/_utilities.scss

```diff
@@ -148,7 +148,7 @@ $utilities: map-merge(
       property: border,
       values: (
         null: var(--#{$prefix}border-width) var(--#{$prefix}border-style) var(--#{$prefix}border-color),
-        "none": $ouds-border-width-none,
+        "none": 0,
```

Comment:
I would by default have kept the "logic" by using `stip-units($ouds-border-width-none)`....?

Reply:
Can't remember your pov is it was replaced by `none` instead of `0` ?

Reply:
FYI, I just tested and `border: 0;` equals `border: none` and not `border-width: 0` when used on border.
Furthermore, Bootstrap doesn't consider 0 as a width neither since it's not in `$border-widths`.